hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/schemas/InitialModServiceConsensusSchema.java

```diff
@@ -71,8 +71,9 @@ public void migrate(@NonNull final MigrationContext ctx) {
             final var topicStoreRef = new AtomicReference<>(ctx.newStates().<TopicID, Topic>get(TOPICS_KEY));
             log.info("BBM: running consensus migration...");
 
-            fs.forEach((k, v) -> {
-                final var pbjTopic = stateToPbj(v);
+            fs.keySet().stream().sorted().forEach(k -> {
```

Comment:
Follow @cody-littley's suggest, use this instead?
fs.treeIterator().setFilter(v -> v instanceof MerkleTopic).forEachRemaining(v -> {

Reply:
yes used `MerkleMapLike.forEachNode`. Does same logic as far as I see